src/ice/pair.rs

```diff
@@ -114,13 +112,12 @@ impl Default for PairId {
 }
 
 impl CandidatePair {
-    pub fn new(local_idx: usize, remote_idx: usize, prio: u64, timing_config: StunTiming) -> Self {
+    pub fn new(local_idx: usize, remote_idx: usize, prio: u64, timing_config: &StunTiming) -> Self {
         CandidatePair {
             local_idx,
             remote_idx,
             prio,
             binding_attempts: VecDeque::with_capacity(timing_config.max_retransmits() + 1),
```

Comment:
What if we reintroduce the `STUN_MAX_RETRANS` constant (it's still referenced in doc/comments I realize), and do this capacity as `2 * STUN_MAX_RETRANS`. That way we can avoid needing the timing config for the constructor.

Reply:
You mean with the default value? Because the max retransmissions are configurable now.

Reply:
Yeah, so the 9 goes back into a constant, and it's used from the configuration.

This call site can use 2 * 9 which means _most_ configurations would never incur an extra allocation here due to the capacity being too low.